psftransltator.py

```diff
@@ -1,13 +1,17 @@
+#!/usr/bin/env python
```

Comment:
This means you can type `./psftranslator.py` instead of `python psftranslator.py`